tapdb/sqlutils_test.go

```diff
@@ -221,11 +221,15 @@ func (d *DbHandler) AddRandomServerAddrs(t *testing.T,
 	return addrs
 }
 
-// NewDbHandle creates a new store and query handle to the test database.
-func NewDbHandle(t *testing.T) *DbHandler {
-	// Create a new test database.
-	db := NewTestDB(t)
+// DbStore is a database store interface.
+type DbStore interface {
```

Comment:
I think instead of using yet another interface, we can just use `*BaseDB` for functions that need a generic handling of database related things. See https://github.com/lightninglabs/taproot-assets/blob/6a18d307e171d46180edf40d73f466bb514a23f5/tapdb/asset_minting_test.go#L45 for example.

Reply:
I don't think we can use `*SqliteStore` as the type `*BaseDB`?

Reply:
Not directly, no. But `SqliteStore` has the `BaseDB` embedded.

Reply:
Ah thanks for pointing that out, that works.